planner/core/common_plans.go

```diff
@@ -835,6 +835,9 @@ func (e *Explain) RenderResult() error {
 		hints := GenHintsFromPhysicalPlan(e.TargetPlan)
 		hints = append(hints, hint.ExtractTableHintsFromStmtNode(e.ExecStmt, nil)...)
 		e.Rows = append(e.Rows, []string{hint.RestoreOptimizerHints(hints)})
+	case ast.ExplainFormatVerbose:
```

Comment:
can we remove this feature since it's not supported and we are not going to support it in the near future?